sklearn/tests/test_config.py

```diff
@@ -0,0 +1,26 @@
+from sklearn import get_config, set_config
+from sklearn.utils.testing import assert_equal
+
+
+def test_set_config():
+    assert_equal(get_config(), {'assume_finite': False})
+
+    # Not using as a context manager affects nothing
+    set_config(assume_finite=True)
```

Comment:
Hmm. I need to check how numpy's `seterr` operates, because apparently global state is changed when not used as a context manager.